### DISCUSSION TITLE
How to reduce the consumption of RAM?

### DISCUSSION THREAD

**Discussion by testinghere57564** [qdm12/gluetun#828] [Q&A]
Good afternoon, I use the commands -е DOT_CACHING=off -e BLOCK_MALICIOUS=off when starting a container, my container consumes 35mb.![image](https://user-images.githubusercontent.com/91505572/151668143-29dd289c-1a9b-478b-a27a-e75d54eea526.png) However, the final consumption figure in the task manager is several times higher than what it should be. I have noticed that running docker vmmem without options consumes 2gb of ram.![image](https://user-images.githubusercontent.com/91505572/151668120-1bd58240-2e1b-4dfc-ba83-c9246b3c0ba4.png) In this case, I have 84 containers running, which should be 3000 mb. Even with the standard 2 GB running docker, 5 GB of RAM should be wasted, not 8 GB. ![image](https://user-images.githubusercontent.com/91505572/151668114-cab96363-dfb4-4e0b-8507-067bb98845a5.png)All containers run beyond the standard 2 GB. I do not know what to do.
Is it possible not to uselessly fill 2 GB, because containers are launched outside of them and is it possible to reduce CPU and memory consumption? This is how my CPU graph looks like, at this time I am not using connections, but only connected to them.
![image](https://user-images.githubusercontent.com/91505572/151668423-ff21a1a1-d666-42e2-a43e-f3e15d47d02a.png)
Thanks.

**Comment by qdm12** [comment]
1. Vmmem is a virtual machine of some sort, so it'll always use more memory than all your containers memory
2. Try to limit the memory usage of wsl/wsl2. Both tend to cache disk on ram and will use all the ram available (which is stupid). Example how to limit it: https://superuser.com/a/1635609
3. For CPU I have no idea. How many cores do you have, what percentage is vmmem using?
